org.eclipse.january.examples/src/org/eclipse/january/examples/dataset/SlicingExamples.java

```diff
@@ -41,6 +43,90 @@
 	}
+	
+	/**
+	 * Slice using basic 1D int array. The goal is to take the data from index 1
+	 * @throws Exception 
+	 */
+	@Test
+	public void sliceFrom1DFromIndex1() throws DatasetException {
+		final Dataset onedData = DatasetFactory.createFromObject(new int[]{1,2,3});
+		Dataset sliceData = onedData.getSlice(new Slice(1, null, null));
+		System.out.println(sliceData.toString(true));
+	}
+	
+	/**
+	 * Slice using basic 1D int array. The goal is to create a Slice with only the stop index
+	 * @throws Exception 
+	 */
+	@Test
+	public void sliceFrom1DWithStopIndex() throws DatasetException {
+		final Dataset onedData = DatasetFactory.createFromObject(new int[]{1,2,3});
+		Dataset sliceData = onedData.getSlice(new Slice(2));
+		System.out.println(sliceData.toString(true));
+	}
+	
+	
+	/**
+	 * Slice using basic 1D int array. The goal is to get the middle of the array
+	 * @throws Exception 
+	 */
+	@Test
+	public void sliceFrom1D() throws DatasetException {
+		final Dataset onedData = DatasetFactory.createFromObject(new int[]{10,9,8,7,6,5,4,3,2,1,0});
+		//we want to get the 3 values middle of the array, that's to say 6,5,4
+		//This mean that you get the value at the 4th position to the value at the 6th position because the second one is exclusive, so you get at the index of end-1
+		//and apply it to the Dataset, this will return the new Dataset :
+		Dataset sliceData = onedData.getSlice(new Slice(4, 7));
+		//and show the informations :
+		System.out.println(sliceData.toString(true));
+	}
+	
+
+	/**
+	 * Slice using basic 1D int array. The goal is to get from the middle of the array to the end
+	 * @throws Exception 
+	 */
+	@Test
+	public void sliceFrom1DGettingEnd() throws DatasetException {
+		final Dataset onedData = DatasetFactory.createFromObject(new int[]{10,9,8,7,6,5,4,3,2,1,0});
+		//we wanted to get the values from the middle, to the end-1, that's to say from 5 to 1
+		//to do that, we can do a simple slice like that :
+		Slice getter = new Slice(5, -1);
+		//This mean that you get the value from the 5th position to the end, because -1 like in python represent the end here it will not get the 0 because the end is exclusive.
+		//and apply it to the Dataset, this will return the new Dataset :
+		Dataset sliceData = onedData.getSlice(getter);
+		//and show the informations :
+		System.out.println(sliceData.toString(true));
+	}
+	
+	/**
+	 * Slice using basic 2D int array. The goal is to get the middle of the array
+	 * @throws Exception 
+	 */
+	@Test
+	public void sliceFrom2D() throws DatasetException {
+		final Dataset twodData = DatasetFactory.createFromObject(new int[][]{	{1,2,3,4},
+																				{4,5,6,7},
+																				{7,8,9,10},
+																				{11,10,11,10}});
+		//we want to get the 4 values middle of the array, that's to say :	5,6
+																		// :	8,9
+		//to do that, we need to do a first slice for the first dimension, that's to say the rows :
+		Slice getterFirstD = new Slice(1,3);
```

Comment:
Can you update this example so that the two slices are different values please? e.g. return {6, 7}, {9, 10} instead.